### ISSUE TITLE
Job Setup Page

### ISSUE THREAD

**Issue by makdaddy8888** [MicrosoftDocs/dynamics365smb-docs#702]
can we please get some more details around the 
Auotmatic Update Job Item Cost
Apply Usage link by Default
Allow Budget/Billable Lines Def

what do they do and why would you disable some of these functions?

Thank you
[Enter feedback here]

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b7d0e112-8f96-e1b0-ce39-4998bc7db75e
* Version Independent ID: 5fe8db51-1f8b-2a0b-a22a-f243448ac043
* Content: [Project Management - Business Central](https://docs.microsoft.com/en-au/dynamics365/business-central/projects-manage-projects)
* Content Source: [business-central/projects-manage-projects.md](https://github.com/MicrosoftDocs/dynamics365smb-docs/blob/main/business-central/projects-manage-projects.md)
* Service: **dynamics365-business-central**
* GitHub Login: @edupont04
* Microsoft Alias: **edupont**

**Comment by edupont04** [comment]
Hi Ben,
Our project management content can be hard to follow because the module itself is rather complex. The fields that you are asking about are described to some detail in the tooltips, but of course there is more to say, and we have a work item in the backlog for someone to help us build out the content.
The article for how to set up jobs does provide a bit more context: https://docs.microsoft.com/dynamics365/business-central/projects-how-setup-jobs
